README.md

```diff
@@ -83,6 +83,12 @@ require 'quality/rake/task'
 Quality::Rake::Task.new
 ```
 
+Make sure you're including `Quality::Rake::Task` only in the
+same environment you've installed the `quality` gem. For example,
+if you're using Rails you may want to check `Rails.env.development?`
+before the require, since Rails uses it's current environment
+as a group for the `Bundler`.
+
```

Comment:
If you're using Rails, I guess it makes sense to put the quality gem into the development group in Gemfile and then add guards. I don't use Rails a lot, but I assume the reasoning is that it shortens deployment cycles not to have to install extra gems in envs where you don't need them, right, and people use Rakefiles for tasks in production?

If you're not using Rails, looks like the above config works fine for both files, so long as you don't run `bundle install --deployment` and then expect Rakefile to still work.

Do I understand it correctly?

Reply:
Yes, I agree it makes sense to put it into the `:development` group.

But, people actually use `rake` in production for many reasons. For example, the `rake db:migration` during a deployment.

So as it will not be installed over there, an exception will occur (that's what happened to me).

Reply:
Cool, makes sense!  Since there's two ways of doing things depending on whether you're using Rails, how about we put an alternate example for Rails after the original non-Rails example?  Might eliminate some or all of the need for the explanation by just showing how things are done...